app/main.py

```diff
@@ -1 +1,5 @@
-# write your code here
+def copy_file(command: str) -> None:
+    elements = command.split()
```

Comment:
Check command length, it should equal exactly to 3 arguments
Then use unpacking, for example:
a, b, c = [1,2,3]
This way you'll avoid accessing command items using indexes